doc/user/metrics/README.md

```diff
@@ -91,3 +110,4 @@ The operator uses [Prometheus][prometheus] to expose a number of metrics by defa
 [ownerref-permission]: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#ownerreferencespermissionenforcement
 [ksm]: https://github.com/kubernetes/kube-state-metrics
 [controller-metrics]: https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/internal/controller/metrics
+[kube-prometheus]: https://github.com/coreos/kube-prometheus
```

Comment:
I think we should link here:
```suggestion
[prometheus-operator]: https://github.com/coreos/prometheus-operator
```

Reply:
I need use these days and the  https://github.com/coreos/prometheus-operator was not working when we follow the steps then I spoke with its maintainers they said that they are no longer really maintainer and using it and suggested me use the https://github.com/coreos/kube-prometheus which actually as very straight forward.  

 think we should update all links. What do you think?